roboflow/core/project.py

```diff
@@ -250,7 +255,7 @@ def upload(
         """upload function
         :param image_path: path to image you'd like to upload
         :param annotation_path: if you're upload annotation, path to it
-        :param hosted_image: whether the image is hosted
+        :param is_hosted: whether the image is hosted
```

Comment:
we can kill this docstring cause `is_hosted` gets defined at the beginning of this function rather than being pulled from the function `kwargs`.

Reply:
i almost wonder if we should create a helper method to determine this in `util/image_utils.py` or a new utility file, like `util/file_utils.py`... that would allow the logic to be easily reusable, updatable, testable, and also make it so that we don't have to pass this kwarg around. e.g:

```
def is_hosted(path):
    return image_path.startswith("http://") or image_path.startswith(
        "https://"
    )
```

just a thought.

Reply:
better way imho

```
is_hosted = not Path(<image_path).exists()

```